src/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { MapOptions } from 'leaflet'
+import { getCrsRd } from './utils'
+
+// eslint-disable-next-line import/prefer-default-export
+export const DEFAULT_AMSTERDAM_MAPS_OPTIONS = {
```

Comment:
I think we can use a normal type here instead of casting it. I've done the following locally and I am not getting any error:

```ts
export const DEFAULT_AMSTERDAM_MAPS_OPTIONS: MapOptions = {
  ...
}
```